### PR TITLE
Updated SDC Troubleshooting

### PR DESCRIPTION
**Pull request by rishabhatdell**
# Description
Updating SDC troubleshooting for OpenShift 4.16

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by shanmydell — CHANGES_REQUESTED** [review]
Hi Rishab, 

can we get our upgrading operator section updated with this info

thanks

**Comment by adarsh-dell** [comment]
@rishabhatdell Do we still need this PR?

File: content/docs/csidriver/troubleshooting/powerflex.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: Troubleshooting PowerFlex Driver
 
 | Symptoms | Prevention, Resolution or Workaround |
 |------------|--------------|
-| After installation vxflexos-node pods are in an `Init:CrashLoopBackOff` state in OpenShift 4.16 with error message: ```Back-off restarting failed container sdc in pod vxflexos-node``` on non-supported kernel versions. | Use SDC version 4.5.2.1 in OpenShift 4.16. |
+| After installation vxflexos-node pods are in an `Init:CrashLoopBackOff` state in OpenShift 4.16 with error message: ```Back-off restarting failed container sdc in pod vxflexos-node``` on non-supported kernel versions. | Re-pull the latest SDC 4.5.2.1 Image in OpenShift 4.16. |
```
Comment (gallacher) [inline content/docs/csidriver/troubleshooting/powerflex.md:9]: Add comma.

"After installation, vxflexos-node pods..."

**Comment by rishabhatdell** [comment]
Closing this PR as the issue has been fixed in the CSM 1.11.1 release.

**Pull request closed without merging**